app/models/views/summary.rb

```diff
@@ -34,6 +35,12 @@ def initialize(session)
       end
     end
 
+    def fee_paid
+      fee = I18n.t("fee_paid_#{@fee_paid}", scope: 'summary')
+      fee += ", on #{Date.parse(@fee_date_paid).strftime('%e %b %Y')}" if @fee_date_paid
+      fee
```

Comment:
Rather than concatenating strings, I'd do something like:

```ruby
def fee_paid
  message = I18n.t("fee_paid_#{@fee_paid}", scope: 'summary')
  "#{message}#{payment_date}"
end

def payment_date
  format = '%e %b %Y'
  date = ", on #{Date.parse(@fee_date_paid).strftime(format)}" 
  @fee_date_paid ? date : ''
end
```